docs/src/community/goversance.md

```diff
@@ -0,0 +1,34 @@
+# Code of Conduct
+
+Youki follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md).
+
+# Maintainership
+
+Youki is governed by Maintainers who are elected from active contributors.
+Youki will keep the [vendor-neutrality](https://contribute.cncf.io/maintainers/community/vendor-neutrality/).
+Maintainers are [here](./maintainer.md).
+
+## Roles
+
+Maintainers are consist of the following roles:
+
+- Committer (Full maintainership)
+    Committers have full write accesses to repos.
+    Committers’ commits should still be made via GitHub pull requests (except for urgent security fixes), and should not be pushed directly.
```

Comment:
Agree with @yihuaf. If we only have one role for now why do we make the distinction between maintainers and commiters?
```suggestion

- Maintainer
    Maintainer have full write accesses to repos.
    Maintainer's commits should still be made via GitHub pull requests (except for urgent security fixes), and should not be pushed directly.
```
If we decide on more roles in the future then we can make that distinction but for now it is a source of potential confusion because readers will wonder what the difference is.